src/dimension/DimensionSQLWriter.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Created by andrelockhart on 5/5/17.
+ */
+import {isRelative, replaceRelative} from '../group/Filter'
+
+// todo - make these sharable thruout app
+const SELECT    = "SELECT ",
+      FROM      = " FROM ",
+      WHERE     = " WHERE ",
+      AND       = " AND ",
+      ORDER_BY  = " ORDER BY ",
+      ASC       = " ASC",
+      DESC      = " DESC",
+      LIMIT     = " LIMIT ",
+      OFFSET    = " OFFSET "
+
+// todo - there is a metric ton of SQL writing fragments that should be organized here
+
+/************   sql fragments from other files/classes   *************************
+  todo
+ */
```

Comment:
delete above todos unless there's more specific actions to take (or create issue)